assets/js/modules/adsense/util.js

```diff
@@ -471,3 +471,22 @@ export const isAdsenseConnectedAnalytics = async() => {
 		resolve( adsenseConnect );
 	} );
 };
+
+/**
+ * Check for any value higher than 0 in values from AdSense data.
+ *
+ * @param {array} data Data returned from the AdSense.
+ * @returns {boolean}
+ */
+export const isDataZeroAdSense = ( data ) => {
```

Comment:
Previously we only looked a the [impressions data](https://github.com/google/site-kit-wp/pull/238/files#diff-1a06a0f16bde2fa02454764097c0a2b1L84). All of the data points on the page except impressions relates to earnings and all might be 0 in any given period (or null for previous periods on new sites), checking the other totals values seems excessive (since any revenue implies impressions). 

Perhaps the value here is catching any non zero value, or if the data shape changes in the future? nice work writing the logic here.